### PR TITLE
Feat: Scope not applicable for city

### PR DESCRIPTION
**Pull request by isaaccodekill**
https://github.com/user-attachments/assets/5f6a05a9-f1e8-4dd6-91b2-d10cdcae5a8d

### CODE REVIEW

**Review by lemilonkh — APPROVED** [review]
Amazing, great work 🙌✨

File: app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx [hunk 1]
```diff
@@ -392,9 +396,9 @@ function SubSectorPage({
             {scopes?.map((scope, index) => (
               <Tab
                 key={index}
-                onClick={() => {
-                  triggerMochLoading();
-                }}
+                // onClick={() => {
+                //   triggerMochLoading();
+                // }}
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx:401]: Yes this was legacy code, that made our application seem much slower.

File: app/src/components/Tabs/Activity/activity-tab.tsx [hunk 2]
```diff
@@ -164,9 +188,39 @@ const ActivityTab: FC<ActivityTabProps> = ({
...
+    setShowUnavailableForm(true);
+  };
+
+  const scopeNotApplicable = useMemo(() => {
+    return inventoryValue?.unavailableExplanation || showUnavailableForm;
+  }, [showUnavailableForm, inventoryValue]);
+
+  const notationKey = useMemo(() => {
+    switch (inventoryValue?.unavailableReason) {
+      case "select-reason-2":
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/components/Tabs/Activity/activity-tab.tsx:213]: yes we can do this. Will add a todo for this

File: app/src/i18n/locales/pt/data.json [hunk 3]
```diff
@@ -755,6 +755,13 @@
   "add-emissions-data-title": "Você não adicionou nenhum dado de emissões",
   "add-emissions-data-subtext": "Adicione dados de emissões e complete seu inventário",
   "add-emission-data-btn": "Adicionar dados de emissões",
+  "reason": "Razão",
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/i18n/locales/pt/data.json:758]: the already exist. On line 46 of this file

File: app/src/util/form-schema/manual-input-hierarchy.json [hunk 4]
```diff
@@ -4222,7 +4222,8 @@
           "id": "direct-measure-wastewater-inside-source-type",
           "options": [
             "source-type-domestic-wastewater",
-            "source-type-industrial-wastewater"
+            "source-type-industrial-wastewater",
+            "source-type-all-wastewater"
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/util/form-schema/manual-input-hierarchy.json:4226]: Good catch.

File: app/src/components/Tabs/Activity/emission-data-section.tsx [hunk 5]
```diff
@@ -199,6 +199,7 @@ const EmissionDataSection = ({
 
   return (
     <>
+      {inventoryValue?.unavailableReason && <p> marked as unexpected </p>}
```
[the remark this answers is not in the document]
Comment (isaaccodekill) [reply app/src/components/Tabs/Activity/emission-data-section.tsx:202]: this was a mistake. Removed now

**Pull request merged**